tracer/test/Datadog.Trace.Debugger.IntegrationTests/ProbesTests.cs

```diff
@@ -290,18 +291,30 @@ async Task RunPhase(LogProbe[] snapshotProbes, ProbeAttributeBase[] probeData, b
 
                 await sample.RunCodeSample();
 
-                var expectedNumberOfSnapshots = DebuggerTestHelper.CalculateExpectedNumberOfSnapshots(probeData);
-                string[] snapshots;
-                if (expectedNumberOfSnapshots == 0)
+                if (DebuggerTestHelper.IsMetricProbe(probeData))
                 {
-                    Assert.True(await agent.WaitForNoSnapshots(), $"Expected 0 snapshots. Actual: {agent.Snapshots.Count}.");
+                    var requests = await agent.WaitForStatsdRequests(probeData.Count(d => d.MetricName != null));
+
+                    foreach (var probeAttributeBase in probeData)
+                    {
+                        requests.Should().Contain(r => r.Contains(probeAttributeBase.MetricName));
```

Comment:
Nit: Maybe we should consider adding more assertions here (as done in: https://github.com/DataDog/dd-trace-dotnet/blob/4a91c86f8bd8e529a4b693109de3c4d18a84524b/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/RuntimeMetricsTests.cs#L108)

Reply:
hmm really I don't know :) I'll add this.

Reply:
Fixed in 1b2566d1a442025260e7614b6c0f95f94133d32f